conf/routes

```diff
@@ -36,6 +36,7 @@ GET     /api2/pluto/projects/:id        controllers.PlutoProjectController.getPr
 POST    /api2/pluto/projects            controllers.PlutoProjectController.createProject()
 PUT     /api2/pluto/projects/:id        controllers.PlutoProjectController.updateProject(id)
 
+GET     /api2/youtube-tag/:id           controllers.YoutubeTagController.getById(id)
```

Comment:
Would `/api2/youtube/content-bundle-tag` be more intention revealing? Also, could this be kept in the `/api2/youtube/` namespace?